### ISSUE TITLE
Create a data generator function to read data from file 

### ISSUE THREAD

**Issue by ellisdg** [ellisdg/3DUnetCNN#1]
A data generator will allow Keras to handle the epochs and learning rate while still reading the data from file.
